tests/integration/targets/cmd_runner/tasks/test_cmd_echo.yml

```diff
@@ -3,17 +3,26 @@
 # GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
```

Comment:
How about adding
```
- debug: var=item
```
here to see what's in there?

Maybe the `when:` in `main.yml` also needs to be changed to `when: item.condition | default(true) | bool` or something due to some issue with old Jinja code (CentOS 6 has a really old one).

Reply:
:facepalm:

Reply:
and it does work with the jinja workaround

Reply:
thanks for the tip

Reply:
I'm glad it worked! I'm really looing forward to get rid of CentOS 6 support ;-)